### PR TITLE
Run tests from installed version.

### PR DESCRIPTION
**Pull request by ocefpaf**
@efiring maybe this change is not desirable but I usually prefer to decouple tests from the installed module and run the tests using the installed version to ensure everything the end user gets will work.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,8 +39,7 @@ install:
 
 script:
     - if [[ $TEST_TARGET == "default" ]]; then
-        python setup.py build_ext -i ;
-        py.test -s -rxs -v gsw ;
+        pushd gsw && py.test -s -rxs -v tests && popd ;
```
Comment (ocefpaf) [inline .travis.yml]: I am no longer building in place and testing. Instead we are using the installed version to run the tests.

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -19,8 +19,13 @@ requirements:
...
+  source_files:
+    - gsw/tests
+  requires:
+    - pytest
   imports:
     - gsw
     - gsw.ice
   commands:
-    - python -c "import gsw; print(dir(gsw))"
+    - cd gsw && py.test -s -rxs -v tests
```
Comment (ocefpaf) [inline conda.recipe/meta.yaml]: With this change it is easier to run the tests on Windows.

File: gsw/tests/test_check_functions.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 import gsw
 from gsw._utilities import Bunch
-from gsw.tests.check_functions import parse_check_functions
+from check_functions import parse_check_functions
```
Comment (ocefpaf) [inline gsw/tests/test_check_functions.py]: The downside is to not create a `tests` submodule.

Comment (efiring) [inline gsw/tests/test_check_functions.py]: I'm not so sure a tests submodule makes sense anyway.

**Pull request merged**